docs/tutorials/adguardhome.md

```diff
@@ -0,0 +1,194 @@
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      serviceAccountName: external-dns
+      containers:
+      - name: external-dns
+        image: registry.k8s.io/external-dns/external-dns:v0.13.5
+        # If authentication is disabled and/or you didn't create
+        # a secret, you can remove this block.
+        envFrom:
+        - secretRef:
+            # Change this if you gave the secret a different name
+            name: adguardhome-password
+        args:
+        - --source=service
+        - --source=ingress
+        # AdGuard Home only supports A/AAAA/CNAME records so there is no mechanism to track ownership.
+        # You don't need to set this flag, but if you leave it unset, you will receive warning
+        # logs when ExternalDNS attempts to create TXT records.
+        - --registry=noop
+        # IMPORTANT: If you have records that you manage manually in AdGuard Home, set
+        # the policy to upsert-only so they do not get deleted.
+        - --policy=upsert-only
+        - --provider=adguardhome
+        # Change this to the actual address of your AdGuard Home web server
+        - --adguardhome-server=http://adguardhome.dns.svc.cluster.local
+      securityContext:
+        fsGroup: 65534 # For ExternalDNS to be able to read Kubernetes token files
+```
+
+### Arguments
+
+ - `--adguardhome-server (env: EXTERNAL_DNS_ADGUARDHOME_SERVER)` - The address of the AdGuard Home web server
+ - `--adguardhome-username (env: EXTERNAL_DNS_ADGUARDHOME_USERNAME)` - The username to the AdGuard Home web server
+ - `--adguardhome-password (env: EXTERNAL_DNS_ADGUARDHOME_PASSWORD)` - The password to the AdGuard Home web server
+ - `--adguardhome-tls-skip-verify (env: EXTERNAL_DNS_ADGUARDHOME_TLS_SKIP_VERIFY)` - Skip verification of any TLS certificates served by the AdGuard Home web server.
+
+## Verify ExternalDNS Works
+
+### Ingress Example
+
+Create an Ingress resource. ExternalDNS will use the hostname specified in the Ingress object.
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: foo
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: foo.bar.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: foo
+            port:
+              number: 80
+```
+
+### Service Example
+
+The below sample application can be used to verify Services work.
+For services ExternalDNS will look for the annotation `external-dns.alpha.kubernetes.io/hostname` on the service and use the corresponding value.
+
+```yaml
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: nginx.external-dns-test.homelab.com
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    name: http
+    targetPort: 80
+  selector:
+    app: nginx
```

Comment:
please change all occurrences of nginx by my-application if you do not need the nginx proxy for a reason